cmd/witness/cmd/verify_test.go

```diff
@@ -17,17 +17,109 @@ package cmd
 import (
 	"bytes"
 	"encoding/json"
+	"io/ioutil"
 	"os"
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/require"
 	"github.com/testifysec/witness/cmd/witness/options"
 	witness "github.com/testifysec/witness/pkg"
 	"github.com/testifysec/witness/pkg/attestation/commandrun"
+	"github.com/testifysec/witness/pkg/cryptoutil"
 	"github.com/testifysec/witness/pkg/dsse"
 	"github.com/testifysec/witness/pkg/policy"
 )
 
+func Test_RunVerifyCA(t *testing.T) {
+	ca, intermediates, leafcert, leafkey := fullChain(t)
+
+	ko := options.KeyOptions{
+		KeyPath: leafkey.Name(),
+		IntermediatePaths: []string{
+			intermediates[0].Name(),
+		},
+		CertPath: leafcert.Name(),
+	}
+
+	caBytes, err := ioutil.ReadFile(ca.Name())
+	require.NoError(t, err)
+
+	policy := makepolicyCA(t, caBytes)
+	signedPolicy, pub := signPolicyRSA(t, policy)
+
+	workingDir := t.TempDir()
+	attestationDir := t.TempDir()
+
+	err = os.WriteFile(workingDir+"signed-policy.json", signedPolicy, 0644)
```

Comment:
make a habit of using filepath.Join for these operations. less error prone and will handle directory separators or lack of them for you.